mdx/api/index.mdx

```diff
@@ -0,0 +1,886 @@
+| `protocolFee` | The amount of ether paid towards the protocol fee (in wei).                                                                                                              |
+| `orders`      | [Signed orders](/docs/api#signed-order) that data is comprised of.                                                                                                       |
+
+### Examples 
+
+#### Simple DAI Buy with WETH
+
+Omit the `takerAddress` to skip `gas` estimation.
+
+<HttpCall fontSize="16px" path="/swap/v0/quote?buyToken=DAI&sellToken=WETH&buyAmount=10000000" />
+
+
+<CodeTabs tabs={['200 - No Taker Address']}>
+
+```json
+{
+  "price": "144.200265328488204418",
+  "to": "0x61935cbdd02287b511119ddb11aeb42f1593b7ef",
+  "data": "0x8bc8efb300000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000989680000000000000000000000000000000000000000000000000000000000000034000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000020000000000000000000000000b0dd0dcada73b715e3b5567ab17bd7f37216a6370000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000410d586a20a4bfffc5000000000000000000000000000000000000000000000000737cad04f84008c100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000005deff8df0000000000000000000000000000000000000000000000000000016ef1607d9a00000000000000000000000000000000000000000000000000000000000001c00000000000000000000000000000000000000000000000000000000000000220000000000000000000000000000000000000000000000000000000000000028000000000000000000000000000000000000000000000000000000000000002800000000000000000000000000000000000000000000000000000000000000024f47261b00000000000000000000000006b175474e89094c44da98b954eedeac495271d0f000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024f47261b0000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000421c3046cceaabdc8efe55f92bff5d96aacfa46c144bfc9e1c13beb5074651ef658661f5e9af72e3c708aba61c00f535f7ff0364d3fdf8985b14b367d1bc8b139b4c03000000000000000000000000000000000000000000000000000000000000",
+  "value": "1200000",
+  "gasPrice": "8",
```

Comment:
I'd update this with the latest response, a number of fields have been changed or removed.

```json
{
  "price": "166.933760683760683761",
  "to": "0x61935cbdd02287b511119ddb11aeb42f1593b7ef",
  "data": "0x8bc8efb30000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000098968000000000000000000000000000000000000000000000000000000000000003c000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000020000000000000000000000000b0dc61047847732a013ce27341228228a38655a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000001100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000b71b0000000000000000000000000000000000000000000000000000000000000118cc0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000967a67f0ab52fa0df98c6823c36b346e4bb12576f6c02a6becded7a7572f530e7d5b97de00000000000000000000000000000000000000000000000000000000000001c000000000000000000000000000000000000000000000000000000000000002a00000000000000000000000000000000000000000000000000000000000000300000000000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000000000000000000000000000000000a4dc1600f30000000000000000000000006b175474e89094c44da98b954eedeac495271d0f000000000000000000000000b0dc61047847732a013ce27341228228a38655a000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000020000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000024f47261b0000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc20000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000010400000000000000000000000000000000000000000000000000000000000000",
  "value": "1200000000000000",
  "gasPrice": "8000000000",
  "protocolFee": "1200000000000000",
  "makerAssetAmount": "10000000",
  "totalTakerAssetAmount": "59904",
  "orders": [
    {
      "chainId": 1,
      "exchangeAddress": "0x61935cbdd02287b511119ddb11aeb42f1593b7ef",
      "makerAddress": "0xb0dc61047847732a013ce27341228228a38655a0",
      "takerAddress": "0x0000000000000000000000000000000000000000",
      "feeRecipientAddress": "0x1000000000000000000000000000000000000011",
      "senderAddress": "0x0000000000000000000000000000000000000000",
      "makerAssetAmount": "12000000",
      "takerAssetAmount": "71884",
      "makerFee": "0",
      "takerFee": "0",
      "expirationTimeSeconds": "2524604400",
      "salt": "77492104380396904871793237448137396496258875317051189624210465011432762873822",
      "makerAssetData": "0xdc1600f30000000000000000000000006b175474e89094c44da98b954eedeac495271d0f000000000000000000000000b0dc61047847732a013ce27341228228a38655a000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000020000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
      "takerAssetData": "0xf47261b0000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
      "makerFeeAssetData": "0x",
      "takerFeeAssetData": "0x",
      "signature": "0x04"
    }
  ]
}

```